Microsoft.Toolkit.Uwp.UI.Controls/Menu/Menu.Events.cs

```diff
@@ -198,16 +200,52 @@ private void Menu_Loaded(object sender, RoutedEventArgs e)
                 _wrapPanel.Orientation = Orientation;
             }
 
+            Window.Current.CoreWindow.PointerMoved -= CoreWindow_PointerMoved;
             LostFocus -= Menu_LostFocus;
             LostFocus += Menu_LostFocus;
             Dispatcher.AcceleratorKeyActivated -= Dispatcher_AcceleratorKeyActivated;
             Window.Current.CoreWindow.KeyDown -= CoreWindow_KeyDown;
             Dispatcher.AcceleratorKeyActivated += Dispatcher_AcceleratorKeyActivated;
             Window.Current.CoreWindow.KeyDown += CoreWindow_KeyDown;
+            Window.Current.CoreWindow.PointerMoved += CoreWindow_PointerMoved;
+        }
+
+        private void Menu_LayoutUpdated(object sender, object e)
```

Comment:
[LayoutUpdated](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.frameworkelement#Windows_UI_Xaml_FrameworkElement_LayoutUpdated) fires for any changes in the entire tree, not just changes in the Menu, so it will fire much more frequently than OnMenuOpen. I agree with @skendrot, it makes more sense to do this each time the menu is open